hw/ip/keymgr/data/keymgr_testplan.hjson

```diff
@@ -6,31 +6,50 @@
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
+                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson"]
   entries: [
     {
       name: smoke
       desc: '''
-            Basic smoke test accessing a major datapath within the keymgr.
-            Most of other sequences are derived from this to have similar init and sequence.
+            Smoke test accessing a major datapath within the keymgr.
+            Test operations (advance, gen-id and gen-sw-out) in every state
 
             Stimulus:
-            - During init, configure ROM_EXT_DESC_*, SOFTWARE_BINDING_*, Salt_*, key_version.
             - Go through state from `StReset` to `StDisabled`.
-            - Issue SW and HW operation in each state, including invalid operations below
-              - advance state in `StDisabled`.
-              - generate output in states other than normal operating states (`StCreatorRootKey`,
-                `StOwnerIntKey` and `StOwnerIntKey`).
+            - Issue gen-id, gen-sw-output operation in each state, including invalid operations in
+              states other than normal operating states (`StCreatorRootKey`, `StOwnerIntKey` and
+              `StOwnerIntKey`).
```

Comment:
I think the second `StOwnerIntKey` should be `StOwnerRootKey`.

Reply:
Good catch. Fixed. Thanks